src/person/base.py

```diff
@@ -0,0 +1 @@
+# This module implements a class Person to be inherited by other classes
```

Comment:
Let's add an empty line at eof for all files (even for the testing ones), just to avoid weird github stuff